src/main/java/frc/robot/EndgameController.java

```diff
@@ -0,0 +1,195 @@
+package frc.robot;
+
+public class EndgameController {
+    private final EndgameArm m_endgameArmA;
+    private final EndgameArm m_endgameArmB;
+    private double m_ascentSpeed, m_descentSpeed;
+
+    /**
+     * This function will take both EndgameArms as parameters, and will
+     * interface the speed used for climbing.
+     * @param endgameArmA
+     * @param endgameArmB
+     */
+    public EndgameController(EndgameArm endgameArmA, EndgameArm endgameArmB) {
+        m_endgameArmA = endgameArmA;
+        m_endgameArmB = endgameArmB;
+    }
+
+    /**
+     * Sets EndgameArm speeds to speeds previously
+     * configured
+     */
+    public void Ascend() {
+        set(m_ascentSpeed);
+    }
+
+    /**
+     * Sets EndgameArm speeds to speeds previously
+     * configured
+     */
+    public void Descend() {
+        set(m_descentSpeed);
+    }
+
+    /**
+     * Sets the speed of the EndGameController
+     * This will (should) eliminate the need
+     * for speed calibrations
+     * @param speed
+     */
+    public void set(double speed) {
+        if (speed > 0) {
+            setRaising(speed);
+        } else if (speed < 0) {
+            setLowering(speed);
+        }
+    }
+    
+    /**
+     * Configures Default Ascent Speed
+     * @param speed
+     */
+    public void setAscentSpeed(double speed) {
+        m_ascentSpeed = speed;
+    }
+
+    /**
+     * Configures the Default Descent Speed
+     * @param speed
+     */
+    public void setDescentSpeed(double speed) {
+        m_descentSpeed = speed;
+    }
+
+    /**
+     * Hopefully this works lmao
+     * @param speed
+     */
+    private void setRaising(double speed) {
```

Comment:
This is fine for now, but I can work with you to create a single method containing the logic of setRaising and setLowering (since it is almost identical)